.github/workflows/ubuntu_python_matrix_test.yml

```diff
@@ -1,47 +0,0 @@
-name: Ubuntu Pyomo Tests Workflow (Slim, Matrixed, Nightly)
```

Comment:
Did you mean to delete this file?

Reply:
Yes/no. I had removed it originally because I didn't want the Ubuntu tests running EVERY time I messed with the Mac tests. Has been added back.